packages/ckeditor5-ckbox/tests/ckboxuploadadapter.js

```diff
@@ -1137,9 +1137,9 @@ describe( 'CKBoxUploadAdapter', () => {
 						.then( () => {
 							throw new Error( 'Expected to be rejected.' );
 						}, err => {
-							expect( console.error.callCount ).to.equal( 1 );
+							expect( console.error.callCount ).to.equal( 2 );
 							expect( console.error.firstCall.args[ 0 ] ).to.match( /^ckbox-access-default-workspace-error/ );
-							expect( err ).to.equal( 'Cannot access default workspace.' );
+							expect( err ).to.equal( 'Cannot determine a category for the uploaded file.' );
```

Comment:
Why has this changed?

Reply:
Hm, good question. I see that it should not be changed. but I think that `await this._token` might result in forwarding incorrect exception.

Reply:
@niegowski I found an issue. The problem was that the exception should have been thrown before the `try {` block in `_getAvailableCategories`. After the changes, it was thrown inside the block, causing the exception to be caught in the wrong place.